resources/l10n/ja/LC_MESSAGES/loot.po

```diff
@@ -7,289 +7,458 @@ msgid ""
 msgstr ""
 "Project-Id-Version: LOOT 0.10.3\n"
```

Comment:
Please update the Project-Id-Version according to point 5 [here](https://github.com/loot/loot/blob/master/CONTRIBUTING.md#translating-the-loot-application) to `LOOT 0.18.2`.